wiki/Guides/How_To_Get_Your_Map_Ranked/en.md

```diff
@@ -11,11 +11,11 @@ Having trouble getting your [beatmap](/wiki/Beatmaps) ranked? You've come to the
 
-Your map's star priority is important when it comes to moving on to the next step. If a modder particularly likes your map they might give it a kudosu star. The more you work to make your map better, the more likely the modder will give you a kudosu star. You can also gain stars by modding other people's maps and spending the kudosu stars on your own mapset. If you don't think you're very good at modding, practice makes perfect! Small suggestions can really help another mapper out, so don't be afraid to give modding a go.
+Your map's hype train is important when it comes to moving on to the next step. If a modder likes your map, they might give it hype. Hype allows your mod to get noticed more by Beatmap Nominators.
 
 ## Step 2: Calling a Beatmap Nominator
 
-Once your map has been pending for a while, has a relatively high star priority, and you think it's perfect and ready to be ranked, it could be time to get a [Beatmap Nominator](/wiki/People/Beatmap_Nomination_Group). Make sure you check the team member's signature or userpage to see whether they have a modding policy; if they're open to requests or there's nothing there, ask them politely in game if they wouldn't mind taking a look at your map. Lots of BN are extremely busy — there are an awful lot of pending maps to get through, so don't be too disheartened if you are turned down. Having plenty of mods and kudos stars on your map will help encourage BN to look at your map, so if you're having difficulties throwing a few more kudos stars your map's way might improve your chances. Once a BN has modded your map and you've made the fixes necessary, call them back to take another look and see whether it is worthy of a bubble.
+Once your map has been pending for a while, has a relatively high hype train, and you think it's perfect and ready to be ranked, it's time to get a [Beatmap Nominator](/wiki/People/Beatmap_Nomination_Group). Make sure you check the team member's signature or userpage to see whether they have a modding policy; if they're open to requests or there's nothing there, ask them politely in game if they wouldn't mind taking a look at your map. Lots of BN are extremely busy — there are an awful lot of pending maps to get through, so don't be too disheartened if you are turned down. Having plenty of mods and kudos stars on your map will help encourage BN to look at your map, so if you're having difficulties throwing a few more kudos stars your map's way might improve your chances. Once a BN has modded your map and you've made the fixes necessary, call them back to take another look and see whether it is worthy of a bubble.
```

Comment:
It only needs 5 hype before it can get nominated.

Reply:
Fixed